classport-agent/src/test/java/ClassportAgentTest.java

```diff
@@ -0,0 +1,164 @@
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.lang.annotation.Annotation;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.nio.file.Files;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import org.junit.jupiter.api.Test;
+
+import io.github.chains_project.classport.agent.ClassportAgent;
+import io.github.chains_project.classport.commons.AnnotationReader;
+import io.github.chains_project.classport.commons.ClassportInfo;
+
+public class ClassportAgentTest {
+
+    private static final String RESOURCES_PATH = "src/test/resources/";
+    private static final String CLASSPORT_TREE_PATH = RESOURCES_PATH + "classport-deps/classport-deps-tree";
```

Comment:
This is error prone because some time you could miss slashes or tests would fail in Windows because they use `\\`. Use `Path.of` and `Path.resolve` APIs. They take care of both. See https://github.com/chains-project/sbom.exe/blob/main/terminator-commons/src/test/java/io/github/algomaster99/terminator/commons/fingerprint/constant_pool/ConstantPoolParserTest.java#L23.